pkg/proxy/proxy_test.go

```diff
@@ -105,6 +105,43 @@ func TestHandleProxyMultiBackend(t *testing.T) {
 	}
 }
 
+func TestLargePayloadOnLimit(t *testing.T) {
+	proxy, err := NewSprayProxy(false, zap.NewNop())
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	w := httptest.NewRecorder()
+	ctx, _ := gin.CreateTestContext(w)
+	ctx.Request = httptest.NewRequest(http.MethodPost, "http://localhost:8080", bytes.NewBuffer(make([]byte, maxReqSize)))
```

Comment:
similar to what we did with client requet timeout, this should be configurable (same env var approach works for me, IMO we have not yet reached "critical mass" where we need to start employing a config map or CRD type for sprayproxy cfg)

Reply:
I considered making that configurable, but it's different from the timeout. I do not expect the GitHub webhook max payload to change that often, if at all. I also do not see us using different values on staging and production cluster. So for the sake of simplicity I decided to hardcode that in code. If people have strong opinion on having this configurable, I'll change it.

Reply:
that's OK .... your explanations are sound

@adambkaplan expressed a similar opinion on our way to get coffee a few minutes ago ;-) 

marking resolved